test/core/WorkflowTestFiles/File Migration/File.dyn

```diff
@@ -1,6 +1,6 @@
 <Workspace Version="0.7.2.2114" X="-92.5" Y="274" zoom="1" Description="" Category="" Name="Home">
   <Elements>
-    <Dynamo.Nodes.CodeBlockNodeModel type="Dynamo.Nodes.CodeBlockNodeModel" guid="75b9d0a3-e954-42b5-8ccf-66845b122e3f" nickname="Code Block" x="462" y="257" isVisible="true" isUpstreamVisible="true" lacing="Disabled" CodeText="c = File.LoadImageFromPath(a);&#xA;File.WriteImage(b,&quot;Image&quot;,c);" ShouldFocus="false" />
+    <Dynamo.Nodes.CodeBlockNodeModel type="Dynamo.Nodes.CodeBlockNodeModel" guid="75b9d0a3-e954-42b5-8ccf-66845b122e3f" nickname="Code Block" x="462" y="257" isVisible="true" isUpstreamVisible="true" lacing="Disabled" CodeText="c = FileSystem.LoadImageFromPath(a);&#xA;FileSystem.WriteImage(b,&quot;Image&quot;,c);" ShouldFocus="false" />
```

Comment:
This is the only way I can get this test to pass.  The migration doesn't handle updating the class renaming in designScript located in a codeblock for the test.  The alternative would be to add an open, save, close, reopen which also should work instead of updating the class name.  The be clear the `WriteImage` node will migrate if placed on the canvas but since it is called in a codeblock it doesn't work upon the first opening.

Reply:
did you check with @aparajit-pratap if type of migration is possible or required?

Reply:
@mjkkirschner we spoke briefly on Friday.  It seems the script in the codeblock itself will not update, however if you save the file and reopen it, codeblocks should update to the latest names.  So that is the alternative.  Maybe that is the better approach to modify the test instead to verify the migration.  Also the file above is v0.7, this shouldn't be an issue for 0.9 and newer

Reply:
actually just tested saving and reopening, doesn't seem to update

Reply:
I don't know what the expectation or functionality is around code block migration - I would rather get this in and file a separate task where we figure that out. Can you file that task?

Reply:
Okay that makes sense, so I should leave it as is so it passes but file a new task explaining the situation?

Reply:
[QNTM-2726](https://jira.autodesk.com/browse/QNTM-2726)

Reply:
@mjkkirschner I just verified the designScript in codeblocks does not update for previous migrations in 1.3 to 2.0.  I have to find a node that has a migration between 0.9 and 1.3 to test older behavior.